### ISSUE TITLE
Change MAXNUMONOFF

### ISSUE THREAD

**Issue by matteobachetti** [scottransom/presto#77]
Hi @scottransom, is there a specific reason why `MAXNUMONOFF` is hardcoded to 40?

For X-ray data from some missions, it would be very helpful to increase it, so that we can exclude from the analysis all the frequent gaps due to telemetry and orbital occultation, SAA passages, etc, and get more consistent statistics.

(I'm tagging here also @ferrigno)

**Comment by scottransom** [comment]
Nope, not at all.  Actually, the ON/OFF pairs are something that I think is rarely if ever used in PRESTO.  If you use it, I'd happily accept a patch to make that dynamic (or have a higher fixed number).

**Comment by tanumanghosh** [comment]
Hi, @scottransom, and @matteobachetti, was this issue resolved? Is it possible to change the number for MAXNUMONOFF in the current PRESTO version?
